### DISCUSSION TITLE
Rtu crashing from adding a induction matrix ident

### DISCUSSION THREAD

**Discussion by Overhandmite** [MikaylaFischler/cc-mek-scada#311] [Support]
I added a induction matrix ident into the rtu config and it won’t boot up now 
![Uploading image.jpg…]()

**Comment by MikaylaFischler** [comment]
I think you posted that before GitHub managed to upload the image, can you send it again?

**Comment by MikaylaFischler** [comment]
Per your comment on #269 it sounds like you either have a missing comma, missing (or too many) brackets, or a string that doesn't have both quotes

**Comment by MikaylaFischler** [comment]
Issue was due to missing commas.
